ros_ws/src/autonomous/driving/wallfollowing5/src/speed_controller.cpp

```diff
@@ -11,38 +11,39 @@ SpeedController::SpeedController()
 
 double SpeedController::calcMaxCurveSpeed(double radius)
 {
-    return sqrt(PhysicalProperties::DYNAMIC_FRICTION * 9.81 * radius);
+    return sqrt(PhysicalProperties::getDynamicFriction() * 9.81 * radius);
 }
```

Comment:
Lots of duplication here, can this be improved?

Reply:
I don't know how this could be improved while it has to call this function to get always the updates from the parameter server and it doesn't look like that a callback function could be used. What I've found is the parameter server the only usefull option to share parameters over several nodes set by dynamic reconfigure.

Reply:
I think now it's more sensible reused